x/incentive/params.go

```diff
@@ -0,0 +1,100 @@
+package incentive
+
+import (
+	fmt "fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const secondsPerDay = 86400
+
+var (
+	defaultMaxUnbondings           = uint32(20)
+	defaultUnbondingDurationShort  = uint64(secondsPerDay)
+	defaultUnbondingDurationMiddle = uint64(secondsPerDay * 7)
+	defaultUnbondingDurationLong   = uint64(secondsPerDay * 14)
+	defaultTierWeightShort         = sdk.MustNewDecFromStr("0.5")
+	defaultTierWeightMiddle        = sdk.MustNewDecFromStr("0.8")
+)
+
+// DefaultParams returns a default set of parameters.
+func DefaultParams() Params {
+	return Params{
+		MaxUnbondings:           defaultMaxUnbondings,
+		UnbondingDurationLong:   defaultUnbondingDurationLong,
+		UnbondingDurationMiddle: defaultUnbondingDurationMiddle,
+		UnbondingDurationShort:  defaultUnbondingDurationShort,
+		TierWeightMiddle:        defaultTierWeightMiddle,
+		TierWeightShort:         defaultTierWeightShort,
+	}
+}
```

Comment:
`Params` are deprecated, please use `Msg` based params system. You can seem my draft PR. 
I think, for the sake of time, you can keep this PR without params implementaiton, and we can do that separately. So it will only include params proto.

Reply:
Here I define a `Params` structure and a `MsgGovSetParams`.

No deprecated subspaces are used, so is there any problem with grouping them into a structure called `Params`?

Reply:
It's good. I made the comment at the beginning of the review, and forgot to remove. Thanks for responding though.